consensus/src/network_interface.rs

```diff
@@ -47,6 +48,8 @@ pub enum ConsensusMsg {
     /// ProposalMsg contains the required information for the proposer election protocol to make
     /// its choice (typically depends on round and proposer info).
     ProposalMsg(Box<ProposalMsg>),
+    /// Add here
+    VerifiedProposalMsg(Box<Block>),
```

Comment:
this is a public facing message type, anyone can choose to send such a VerifiedProposalMsg through network, we should just resend through the round_manager_tx intead

Reply:
FIxed.